backend/src/resolvers/posts.js

```diff
@@ -16,6 +16,26 @@ export default {
       )
       session.close()
 
+      return result
+    },
+    UpdatePost: async (object, params, context, resolveInfo) => {
+      const session = context.driver.session()
+      // first delete WROTE relationship and after the update add it again
+      // this is needed because otherwise there would be 2 WROTE relations in the database
+      await session.run(
+        'MATCH (u:User)-[rel:WROTE]->(p:Post) WHERE p.id = $id delete rel', {
+          id: params.id
+        }
+      )
+      const result = await neo4jgraphql(object, params, context, resolveInfo, false)
+      await session.run(
+        'MATCH (u:User), (p:Post) WHERE p.id = $id AND u.slug = $slug CREATE (u)-[:WROTE]->(p)', {
```

Comment:
Three things:
1. If you use `MERGE` instead of `CREATE` that would not create duplicate relationships
2. Use `user.id` not `user.slug` if possible
3. If you setup relationships after you already created the object, use the object. In this case, use `result.id` not `params.id`